requirements.txt

```diff
@@ -1,64 +1,13 @@
-astroid==2.3.3
-attrs==19.3.0
-autopep8==1.4.4
-backcall==0.1.0
-bleach==3.1.4
-cycler==0.10.0
-decorator==4.4.1
-defusedxml==0.6.0
-entrypoints==0.3
-importlib-metadata==0.23
-ipykernel==5.1.3
-ipython==7.9.0
-ipython-genutils==0.2.0
-isort==4.3.21
-jedi==0.15.1
-Jinja2==2.10.3
-json5==0.8.5
-jsonschema==3.1.1
-jupyter-client==5.3.4
-jupyter-core==4.6.1
-jupyterlab==1.2.2
-jupyterlab-server==1.0.6
-kiwisolver==1.1.0
-lazy-object-proxy==1.4.3
-MarkupSafe==1.1.1
-matplotlib==3.1.1
-mccabe==0.6.1
-mistune==0.8.4
-more-itertools==7.2.0
-nbconvert==5.6.1
-nbformat==4.4.0
-notebook==6.0.2
-numpy==1.17.4
-pandas==0.25.3
-pandocfilters==1.4.2
-parso==0.5.1
-pexpect==4.7.0
-pickleshare==0.7.5
-Pillow==6.2.1
-prometheus-client==0.7.1
-prompt-toolkit==2.0.10
-ptyprocess==0.6.0
-pycodestyle==2.5.0
-Pygments==2.4.2
-pylint==2.4.4
-pyparsing==2.4.5
-pyrsistent==0.15.5
-python-dateutil==2.8.1
-pytz==2019.3
-pyzmq==18.1.0
-Send2Trash==1.5.0
-six==1.13.0
-terminado==0.8.2
-testpath==0.4.4
-torch==1.3.1+cpu
-torchvision==0.4.2+cpu
-tornado==6.0.3
-tqdm==4.38.0
-traitlets==4.3.3
-typed-ast==1.4.0
-wcwidth==0.1.7
-webencodings==0.5.1
-wrapt==1.11.2
-zipp==0.6.0
+torch
+jupyterlab
+matplotlib
+seaborn
+tqdm
+pylint
+autopep8
+python-dotenv
+python-dotenv[cli]
+psutil
+lxml
+git+https://github.com/maxjcohen/transformer
```

Comment:
When I said you could add this line locally, i meant it shouldn't be added in the repo's requirements. Indeed, running the benchmark does not require to install or use transformers.

Reply:
Got it!